doc/index.rst

```diff
@@ -432,6 +432,30 @@ Example of returned tag data:
 
         ['Authority in literature', 'Errata']
 
+===================
+Retrieving Version Information
+===================
+
+The Zotero API recommends requesting version information for all (or all changed) items and collections when implementing syncing.  The following convience methods simplify this process.  Note that by default these methods impose no limit on the number of items/collection versions returned.  To retrieve only the versions of items changed after a given version use the since search/request parameter.
```

Comment:
Can we clarify what the since / search parameter is? I think it should be rendered `as code`, and include a link to https://www.zotero.org/support/dev/web_api/v3/syncing